fastlane/Fastfile

```diff
@@ -105,18 +111,44 @@ platform :ios do
     )
   end
 
-  private_lane :lint do |options|
+  private_lane :lint_podspec do |options|
     pod_lib_lint(sources: [options[:source], "https://github.com/CocoaPods/Specs"], allow_warnings: true, verbose: false)
   end
 
   desc 'Lint the podspec on STG spec repo'
   lane :stg_lint do
-    lint(source: ENV['REM_FL_STG_SPEC_REPO'])
+    lint_podspec(source: ENV['REM_FL_STG_SPEC_REPO'])
   end
 
   desc 'Lint the podspec on PROD spec repo'
   lane :prd_lint do
-    lint(source: ENV['REM_FL_SPEC_REPO'])
+    lint_podspec(source: ENV['REM_FL_SPEC_REPO'])
+  end
+
+  lane :lint_module do
```

Comment:
if we call lint_module without any parameter, it means that is_ci = false, right?

Reply:
no, `is_ci` is a built-in fastlane [action](https://docs.fastlane.tools/actions/is_ci/#is_ci)

Reply:
thank you.